src/features/twinClass/selectField/twin-class-select-field.tsx

```diff
@@ -0,0 +1,54 @@
+import { TwinClassContext } from "@/app/twinclass/[twinClassId]/twin-class-context";
+import { ComboboxFormField } from "@/components/form-fields/combobox-form-field";
+import { TextFormItem } from "@/components/form-fields/text-form-field";
+import { TwinClass_DETAILED, useTwinClassSearchV1 } from "@/entities/twinClass";
+import { useContext } from "react";
+import { Control, FieldValues, Path } from "react-hook-form";
+
+type Props<T extends FieldValues> = {
+  control: Control<T>;
+  name: Path<T>;
+  label: string;
+  disabled?: boolean;
+};
+
+export function TwinClassSelectField<T extends FieldValues>(props: Props<T>) {
+  const { twinClass } = useContext(TwinClassContext);
+  const { searchTwinClasses } = useTwinClassSearchV1();
+
+  // NOTE: currently this function is never called
+  // TODO: investigate in context of https://alcosi.atlassian.net/browse/TWINFACES-76
+  async function getById(): Promise<undefined> {
+    return undefined;
```

Comment:
It should be called when the value is passed to this field externally, for example when you have an edit form with this field and the model already contains a value for the field. Then combobox uses this function to fill in the value.